### PR TITLE
Remove deprecated metrics from sys.nodes

### PR DESCRIPTION
**Pull request by mkleen**
## Checklist

 - [x] User relevant changes are recorded in ``CHANGES.txt``
 - [x] Touched code is covered by tests
 - [x] Documentation has been updated if necessary
 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed
 - [x] This does not contain breaking changes, or if it does:
    - It is released within a major release
    - It is recorded in ``CHANGES.txt``
    - It was marked as deprecated in an earlier release if possible
    - You've thought about the consequences and other components are adapted
      (E.g. AdminUI)

### CODE REVIEW

**Comment by mfussenegger** [comment]
Did we double check that the admin-ui isn't using any of those properties anymore?

File: sql/src/main/java/io/crate/expression/reference/sys/node/fs/NodeStatsFsDisksExpression.java [hunk 1]
```diff
@@ -48,12 +48,6 @@
             .put(NodeFsStatsExpression.SIZE, FsInfoHelpers.Path.size(path))
             .put(NodeFsStatsExpression.USED, FsInfoHelpers.Path.used(path))
             .put(NodeFsStatsExpression.AVAILABLE, FsInfoHelpers.Path.available(path))
-
-            .put(NodeFsStatsExpression.READS, -1L)
```
[the remark this answers is not in the document]
Comment (mkleen) [reply sql/src/main/java/io/crate/expression/reference/sys/node/fs/NodeStatsFsDisksExpression.java]: Yes, they are used in another expression. I cleaned it up though.

**Comment by mkleen** [comment]
> Did we double check that the admin-ui isn't using any of those properties anymore?

Good point, I will check. Thank you.

**Comment by mkleen** [comment]
Retest this please

**Pull request merged**